knowledge/animals/birds/black_capped_checkadee/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: hashimhassany
+domain: animals
+seed_examples:
+- answer: North America
+  question: Where does the Black-capped chickadee originate.
+- answer: Paridae family
+  question: This bird is the member of which family?
```

Comment:
This isn't a clear enough question, it should be "What is the black-capped chickadee membership of which family" or the like.